src/main/java/frc/robot/commands/ArcadeDrive.java

```diff
@@ -28,7 +28,8 @@ protected void initialize() {
   @Override
   protected void execute() {
     LogitechController gamepad = Robot.m_oi.getDriverOI();
-    Robot.m_drive.rampedArcadeDrive(gamepad.getLY(),gamepad.getRX());
```

Comment:
We can add logic to check if there's any getRX() value after deadzone checking. If so, used "rampedArcadeDrive()". Otherwise, use "driveStraightGyro()". We can always use "rampedArcadeDrive()" as a placeholder until "driveStraightGyro()" works

Reply:
We would need to also add logic to determine the 1st "driveStraightGyro()" call and set the "targetAngle" only on this call.